### ISSUE TITLE
No @@map created for enums 

### ISSUE THREAD

**Issue by olivierlacan** [iiian/prisma-case-format#44]
This is a fantastic tool, thank you so much for making it.

It worked perfectly on our fields and model names but we did notice that it renamed our enums without creating `@@map` statements so the next `prisma migrate dev` tried to create a migration to rename them in the source database.

```typescript
enum education_level_type {
  BEGINNER
  INTERMEDIATE
  ADVANCED

  @@schema("public")
}
```

became: 

```typescript
enum EducationLevelType {
  BEGINNER
  INTERMEDIATE
  ADVANCED

  @@schema("public")
}
```

Should have been: 
```typescript
enum EducationLevelType {
  BEGINNER
  INTERMEDIATE
  ADVANCED

  @@map("education_level_type")
  @@schema("public")
}
```

**Comment by iiian** [comment]
Hey @olivierlacan, glad you're getting some mileage out of the tool!

Which version of `prisma-case-format` are you using? At this point, my best suggestion is: If you're on `^2.0.0`, go back to `^1.0.0` -- if you're on `^1.0.0` go to `^2.0.0`. Reason being that `v2.0` introduced some new (breaking) semantics around `--enum`s, and I just haven't had enough time to dig through and convince myself it is (or isn't) broken.

I haven't had much mental bandwidth to maintain this project this month, and would probably be able to come back to it ~December, if you're willing to wait.

**Comment by olivierlacan** [comment]
Thanks @iiian, I can confirm that 1.7.3 correctly converts enums from snake_case. 👍

**Comment by iiian** [comment]
> Thanks @iiian, I can confirm that 1.7.3 correctly converts enums from snake_case with a `@@map` statement. 👍 
> 
> 
> 
> However 1.7.3 creates a field `map` for a field named `type` (without renaming it) which doesn't require it.

Yeah 😄 I think I just made the arbitrary decision to always add a redundant map annotation, I think it makes the code easier.

Thanks for reporting this issue, I'll try to get a fix in for v2 next month. Glad this got you around the problem for the time being!

**Comment by iiian** [comment]
Hey @olivierlacan. This issue should be fixed in `prisma-case-format@2.0.2`.

**Comment by iiian** [comment]
Oh, sorry. For better clarity into what happened: I think I made a small miscalculation cutting a new major version--I don't believe I've actually introduced any breaking changes behaviorally. That being said, it should be safe to just jump right ahead to `^2.0.0` if you're running `1.0.0` starting from `2.0.2`.
